pom.xml

```diff
@@ -377,6 +377,7 @@
         <version.org.ow2.asm>6.0</version.org.ow2.asm>
         <version.org.picketbox.picketbox-commons>1.0.0.final</version.org.picketbox.picketbox-commons>
         <version.org.picketlink>2.5.5.SP11</version.org.picketlink>
+        <version.org.powermock>2.0.0-beta.5</version.org.powermock>
```

Comment:
I realize this is for a test, but we're not supposed to use non-final releases as dependencies.

Reply:
This no longer has a pom change.